packages/app/components/media-cropper/index.web.tsx

```diff
@@ -95,7 +95,7 @@ export const MediaCropper = ({
             )}
           />
           <View tw="max-h-[90vh] min-h-[560px]">
-            <View tw="h-[576px] w-full md:w-[480px]">
+            <View tw="h-[60vh] w-full md:w-[480px]">
```

Comment:
Do you want to add a min-height just in case?

Reply:
yeah, that makes sense,  min-height should be the same as the `width`.